cg/apps/demultiplex/sample_sheet/models.py

```diff
@@ -76,6 +78,15 @@ class FlowCellSampleNovaSeq6000Dragen(FlowCellSampleNovaSeq6000):
 class SampleSheet(BaseModel):
     samples: List[FlowCellSample]
 
+    def get_non_pooled_samples(self) -> List[FlowCellSample]:
```

Comment:
Do we want to keep logic outside of our data models? This could go in a utils module as well. We don't have any outspoken design choice regarding this. 

We use Pydantic for most of our domain models, so I think it makes sense to have logic in them as well. It encapsulates logic nicely

Reply:
I think this is well placed 👍